client.go

```diff
@@ -0,0 +1,56 @@
+package compute
+
+import (
+	"io"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+type Client struct {
+	config     *Config
+	httpClient *http.Client
+}
+
+func NewClient(config *Config, httpClient *http.Client) (*Client, error) {
+	if config == nil {
+		return nil, errors.New("failed to NewClient: config cannot be nil")
+	}
+
+	if httpClient == nil {
+		return nil, errors.New("failed to NewClient: httpClient cannot be nil")
+	}
+
+	client := &Client{
+		config:     config,
+		httpClient: httpClient,
+	}
+
+	return client, nil
+}
+
+func NewLocalClient() (*Client, error) {
+	return NewClient(LocalConfig(), http.DefaultClient)
+}
+
+func (c *Client) NewRunnable(environment, customerID, namespace, fnName string) *Runnable {
+	runnable := &Runnable{
+		environment:  environment,
+		customerID:   customerID,
+		namespace:    namespace,
+		functionName: fnName,
+	}
+
+	return runnable
+}
+
+func (c *Client) BuildWith(runnable *Runnable, fn Function) (string, *http.Response, error) {
+	return "", nil, nil
```

Comment:
What's up with this?

Reply:
That's a placeholder! Didn't meant to commit that.